python/vimsence.py

```diff
@@ -260,4 +260,4 @@ def get_directory():
     :returns: string
     '''
 
-    return re.split(r'[\\/]', vim.eval('getcwd()'))[-1]
+    return re.split(r'[\\/]', vim.eval("system('git rev-parse --show-toplevel || pwd')"))[-1]
```

Comment:
This would break on Windows, because there is no `pwd` available in path.

I also think we should not ask Vim to execute a system command. Doing this directly from Python would be a nicer way imo.

Reply:
Thank you for the feedback, I'm gonna try to fix this by using only python